features/rni.feature

```diff
@@ -0,0 +1,187 @@
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En heeft de persoon GEEN 'rni'
+
+    Scenario: niet leveren RNI-deelnemer categorie 7 wanneer er geen opschorting en geen geheimhouding is
+      Gegeven de persoon met burgerservicenummer '000000048' heeft de volgende 'inschrijving' gegevens
+      | naam                            | waarde                                      |
+      | indicatie geheim (70.10)        | 0                                           |
+      | RNI-deelnemer (07.88.10)        | 101                                         |
+      | Omschrijving verdrag (07.88.20) | Belastingverdrag tussen België en Nederland |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000048                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000048 |
+
+    Scenario: wel leveren RNI-deelnemer categorie 7 wanneer er geheimhouding is
+      Gegeven de persoon met burgerservicenummer '000000061' heeft de volgende 'inschrijving' gegevens
+      | naam                            | waarde                                      |
+      | indicatie geheim (70.10)        | 7                                           |
+      | RNI-deelnemer (07.88.10)        | 101                                         |
+      | Omschrijving verdrag (07.88.20) | Belastingverdrag tussen België en Nederland |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000061                       |
+      | fields              | burgerservicenummer             |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 000000061 |
+      | geheimhoudingPersoonsgegevens | true      |
+      En heeft de persoon een 'rni' met de volgende gegevens
+      | naam                   | waarde                                          |
+      | deelnemer.code         | 101                                             |
+      | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+      | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+      | categorie              | Inschrijving                                         |
+
+    Scenario: wel leveren RNI-deelnemer categorie 7 wanneer er opschorting bijhouding is
+      Gegeven de persoon met burgerservicenummer '000000073' heeft de volgende 'inschrijving' gegevens
```

Comment:
Hier geldt hetzelfde als bij de voorgaande 2 scenario's.
Daarnaast denk ik dat de 'En' stap 'En de persoon heeft de volgende 'opschortingBijhouding' gegevens' geintegreerd moet worden met de 'Gegeven' stap 'Gegeven de persoon met burgerservicenummer '000000073' heeft de volgende 'inschrijving' gegevens'.
opschortingBijhouding wordt immers in dezelfde tabel vastgelegd als geheimhouding.

Wil je beide toch scheiden dan zul je 'opschortingBijhouding' moeten toevoegen aan 'tableNameMap' in 'step_defs.js':

    ['opschortingBijhouding', 'lo3_pl'],

Ik weet alleen niet of dat gaat werken aangezien er dan twee namen verwijzen naar de tabel 'lo3_pl'. 'inschrijving' doet dat nl. ook al.

Reply:
alle gegevens uit categorie inschrijving zitten nu in 1 stap, niet meer aparte stap voor opschorting bijhouding